site/content/docs/5.2/forms/layout.md

```diff
@@ -119,6 +126,13 @@ Create horizontal forms with the grid by adding the `.row` class to form groups
 
 At times, you maybe need to use margin or padding utilities to create that perfect alignment you need. For example, we've removed the `padding-top` on our stacked radio inputs label to better align the text baseline.
 
+<!-- Boosted mod : design callout -->
+{{< ods-incompatibility-alert >}}
+This form variant can not be used because it does not respect the Orange Design System specifications.
```

Comment:
Is that because we don't want labels on the left of inputs ? Same as below ?

Reply:
Yes and because the design considers this example as a module and not a unitary component. So, the designer's recommandation is to make developers use unitary form components along with the ODS form templates to build their forms, and not take all the code of this example.

Would it be useful to replace the 1st sentence by this one ?
"This form variant can not be used because it does not respect the Orange Design System specifications. **The form input label should always be placed above the input field.**"

Reply:
Add in the message that it's because of the labels